checkers/testdata/boolExprSimplify/negative_tests.go

```diff
@@ -47,3 +47,16 @@ func floatCompare() {
 	// Can't be simplified to `f1 != f2`.
 	_ = !(f1 == f2)
 }
+
+func balancedIncDec(x, y, z int) {
+	// This is usually done on purpose.
```

Comment:
What purpose? 🤔 Just curious.

Reply:
Making expressions in a row (multi-statement) look more beautiful.
This is something like "balancing" them.
Same reason why someone writes +0 in expressions:
```go
x+0 < a+0
x+1 < b+1
x+2 < b+2
```

Reply:
Ah, context.. thanks for the clarification.